flink-connectors/flink-connector-hive/src/test/java/org/apache/flink/table/catalog/hive/HiveCatalogTest.java

```diff
@@ -120,4 +120,23 @@ public void testRetrieveFlinkProperties() throws Exception {
         assertEquals(
                 hiveTable.getOptions().get("flink.url"), "jdbc:clickhouse://host:port/testUrl2");
     }
+
+    @Test
+    public void testGetNoSchemaGenericTable() throws Exception {
+        ObjectPath hiveObjectPath =
+                new ObjectPath(HiveCatalog.DEFAULT_DB, "testGetNoSchemaGenericTable");
+
+        Map<String, String> properties =
+                new HashMap<>(new FileSystem().path("/test_path").toProperties());
```

Comment:
Why we need these properties?

Reply:
Because in the lastest code, the table that not have properties 'connector' will be define as a hive table but not a GenericTable. And the hive table can be drop when it is no schema.

Reply:
But you're already setting 'connector' to jdbc right?

Reply:
In the new commit remove unnecessary properties.